src/index.js

```diff
@@ -0,0 +1,35 @@
+import Weather from './weather';
+import Storage from './storage';
+import Dom from './dom';
+
+const storage = new Storage();
+
+const dom = new Dom();
+
+const storedWeather = storage.getLocationData();
+
+const weather = new Weather(storedWeather.city, storedWeather.unit);
+
+const changeBtn = document.getElementById('w-change-btn');
+const city = document.getElementById('city');
+const unit = document.getElementById('unit');
+
+function getWeather() {
+  weather.getWeather(city.value, unit.value)
+    .then(results => {
+      dom.updateDom(results);
+    });
+  // eslint-disable-next-line no-alert
```

Comment:
I don't think we need this line comment anymore.
It would be nice to pop-up a div message whenever a user types in the wrong location.

**I trust you can do this before you merge**